packages/webapp/public/locales/en/translation.json

```diff
@@ -1952,7 +1952,7 @@
     "ADD_TASK_FLOW": "task creation",
     "AMOUNT_TO_ALLOCATE": "Amount to allocate",
     "ANIMAL_MOVING_TO_LOCATION": "Moving to:",
-    "ANIMAL_MOVEMENT_EXPANDING_SUMMARY_TITLE": "See detail list of animals to move",
+    "ANIMAL_MOVEMENT_EXPANDING_SUMMARY_TITLE": "Animals in this task",
```

Comment:
The original string had just gone up to CrowdIn less than an hour ago, so I went ahead and _manually adjusted_ this on CrowdIn (so translators will not have to translate the now irrelevant string).

Since this is already on CrowdIn, I will remove the "new translations" tag from this branch.